spec/src/main/asciidoc/architecture.adoc

```diff
@@ -182,15 +182,19 @@ to `true` and have multiple methods annotated as metric that have the same metho
 .Example of reused counters
 [source,java]
 ----
-    @Counted(name = "countMe", absolute = true, reusable = true)
+    @HitCounted(name = "countMe", absolute = true, reusable = true)
     public void countMeA() { }
 
-    @Counted(name = "countMe", absolute = true, reusable = true)
+    @HitCounted(name = "countMe", absolute = true, reusable = true)
     public void countMeB() { }
 ----
 
 In the above examples both `countMeA()` and `countMeB()` will share a single Counter with registered name `countMe` in application scope.
 
+The implementation must throw an 'IllegalArgumentException' if a counter defined via `@HitCounted` is reused as `@ParallelCounted` or vice versa.
```

Comment:
Isn't this just the same as reusing the same metric name for `@Timed` and `@Metered` (or any two metrics with different types?